wire-ios-sync-engine/Source/Utility/Device.swift

```diff
@@ -69,6 +69,10 @@ import Foundation
         case "iPhone14,8":                                  return "iPhone 14 Plus"
         case "iPhone15,2":                                  return "iPhone 14 Pro"
         case "iPhone15,3":                                  return "iPhone 14 Pro Max"
+        case "iPhone15,4":                                  return "iPhone 15"
+        case "iPhone15,5":                                  return "iPhone 15 Plus"
+        case "iPhone16,1":                                  return "iPhone 15 Pro"
+        case "iPhone16,2":                                  return "iPhone 15 Pro Max"
```

Comment:
Where do these strings end up?
If it's only in our logs, then there's no need for maintaining this list. We can always do it when we analyse logs.
If it's relevant for the user, it makes sense.

Reply:
I completed the existing list from a github repo

Reply:
it's in the body of the mail report